### ISSUE TITLE
The field option for ordering by dates in blueprint doesn't have any effect

### ISSUE THREAD

**Issue by pmhoudry** [getkirby/kirby#245]
Parent blueprint::

    pages:
      num: date
      field: added

Child blueprint::

    added:
      type: date

Result : in the panel the ordering date is displayed as 1970/01/01, and the ordering doesn't work.

Solution : call the field 'date'. That way the ordering works, even if I try to order by another datefield.

**Comment by distantnative** [comment]
I'll try to look into this a lil closer within the next days, but just from a having a glimpse at the code: Could you try the following to see if it works?

```
pages:
  num:
    mode: date
    field: added
```

**Comment by distantnative** [comment]
Just did some testing and this is the way which works:

```
pages:
  num:
    mode: date
    field: added
```

I will open a new issue for changing the documentation to include this.

**Comment by pmhoudry** [comment]
Ok great, thank you. There is a thread regarding this issue in the forum, I'll update it with this info.

-----Message d'origine-----
De : "Nico Hoffmann" <notifications@github.com>
Envoyé : ‎05/‎05/‎2015 09:51
À : "getkirby/kirby" <kirby@noreply.github.com>
Cc : "pmhoudry" <pmhoudry@gmail.com>
Objet : Re: [kirby] The field option for ordering by dates in blueprintdoesn't have any effect (#245)

Closed #245.
